src/sentry/issues/escalating.py

```diff
@@ -16,15 +16,14 @@
 )
 
 from sentry.models import Group
-from sentry.utils.json import JSONData
 from sentry.utils.snuba import raw_snql_query
 
 QUERY_LIMIT = 10000  # This is the maximum value for Snuba
 # The amount of data needed to generate a group forecast
 SEVEN_DAYS_IN_HOURS = 7 * 24
 
 
-def query_groups_past_counts(groups: List[Group]) -> JSONData:
+def query_groups_past_counts(groups: List[Group]) -> List[Dict[str, Any]]:
```

Comment:
Could we make this a typed dict too? I think we likely know the format of the data here and it's consistent?